demos/common/ota_demo_helpers/ota_demo_helpers.c

```diff
@@ -1,5 +1,5 @@
 /*
- * AWS IoT Device SDK for Embedded C V202009.00
+ * FreeRTOS V202012.00
```

Comment:
FreeRTOS coding standard should  be followed here, can be in another PR

Reply:
Thanks, will create a separate PR for this.